### ISSUE TITLE
Delete Functionality

### ISSUE THREAD

**Issue by Aetherinox** [antoniotejada/obsidian-edit-history#5]
Sorry for such as a simple thing, but is their a feature to delete the history for a certain page? I looked all over, and all I could find is the "Copy" button.

Or do I need to actually go into the directory and find the `edtz` file and delete it manually.

Another request that would be nice, would be a blacklist. I want most of my files to have their edits saved, however, there are 4 or so that I don't want the history tracked on. And right now I'd have to remove the `.md` entry in the whitelist, and manually add all the files I want tracked, and that would be a lot.

Or maybe a frontmatter / metadata value that would make the plugin skip saving edits such as:

```
---
notrack: true
---
```

**Comment by antoniotejada** [comment]
Right, there's no delete from inside Obsidian other than enabling viewing all files in the file explorer and deleting from there. 
There's a TODO in the README.md regarding edit history management. Tagging as enhancement

Regarding blacklist, see #6 and specifically #1, note that the current whitelist does suffix match, not just extension match, so maybe you can find a common suffix used by the files you want to track but not by the other 4.

Out of curiosity, is it really that much of a problem that there are 4 files or so extra being tracked?
